test/metric_value_benchmark/eks_daemonset_test.go

```diff
@@ -66,19 +77,80 @@ func (e *EKSDaemonTestRunner) validateInstanceMetrics(name string) status.TestRe
 	return testResult
 }
 
-func (e *EKSDaemonTestRunner) GetTestName() string {
+func (e *EKSDaemonTestRunner) validateLogs(eks *environment.MetaData) status.TestResult {
+	testResult := status.TestResult{
+		Name:   "emf-logs",
+		Status: status.FAILED,
+	}
+
+	now := time.Now()
+	group := fmt.Sprintf("/aws/containerinsights/%s/performance", eks.EKSClusterName)
+
+	// need to get the instances used for the EKS cluster
+	EKSInstances, err := awsservice.GetEKSInstances(eks.EKSClusterName)
+	if err != nil {
+		log.Println("failed to get EKS instances")
+		return testResult
+	}
+
+	for _, instance := range EKSInstances {
+		validateLogContents := func(s string) bool {
+			return strings.Contains(s, fmt.Sprintf("\"ClusterName\":\"%s\"", eks.EKSClusterName))
+		}
```

Comment:
nit - we could define this function outside of the for loop

Reply:
Sure, but then we would need to pass the environment.Metadata to the function which would then make it so we have to change the MatchEMFLogWithSchema function as it only takes a func(string). Would like to avoid this as it's also used for the ECS tests.

Reply:
Are you sure? I'm looking specifically at this `for _, instance := range EKSInstances`. The validation function here doesn't actually use anything from each instance. It's just validating the cluster name so we could rip it out from here to define once and reuse

Reply:
Aren't you talking about the `validateLogContents` function? If so, this function gets used in `if !awsservice.MatchEMFLogWithSchema(l, rs, validateLogContents) {` and requires the function to be a func(string)

Reply:
Yes. I'm saying you could define it outside of the for loop, pretty sure. But it's not super important so I'm fine just leaving it. Don't want to spend a lot of time on it